background.js

```diff
@@ -31,7 +31,14 @@ app.whenReady().then(() => {
   // TODO: Check if first install and start welcome screen
   // TODO: Check if new version and provide window with details on changes
 
-  cfg = settings.init();
+  try {
+    cfg = settings.init();
+  } catch(err) {
+    gone = true;
```

Comment:
`gone` is declared after this line (line 70 something). `let` will hoist the declaration but not initialize a variable, if I understand it corrently. This can cause problems. Can you move the `let gone = false` above this first use.

Reply:
Done. It was correct, but obtuse. Better to move it to be with the other module-level vars